lib/screenplay/suppressed_predictions.ex

```diff
@@ -38,6 +41,76 @@ defmodule Screenplay.SuppressedPredictions do
     |> Repo.all()
   end
 
+  @doc """
+  Get all suppressed predictions modified for transit data
+  """
+  @spec get_all_suppressed_predictions_transit_data() :: [SuppressedPrediction.t()]
+  def get_all_suppressed_predictions_transit_data do
```

Comment:
nit: "Transit data" is an organizational detail, so probably better to avoid encoding it in the name